x-pack/plugins/observability/server/assets/component_templates/slo_summary_mappings_template.ts

```diff
@@ -55,12 +55,10 @@ export const getSLOSummaryMappingsTemplate = (name: string) => ({
               ignore_above: 256,
             },
             name: {
-              type: 'keyword',
-              ignore_above: 256,
+              type: 'text',
```

Comment:
i will suggest to make this multi field, as keyword and text. Since name can have both kind of use cases.